provisioner/aws-manager.js

```diff
@@ -507,12 +865,11 @@ AwsManager.prototype._trackNewSpotRequest = function (sr) {
   // AWS Api when we submit the SpotRequest
   assert(sr);
 
-  var that = this;
-
   var allKnownSrIds = this.knownSpotInstanceRequestIds();
 
   if (!allKnownSrIds.includes(sr.request.SpotInstanceRequestId)) {
-    var filtered = objFilter(sr.request, that.filters.spotReq);
+    // XXX var filtered = objFilter(sr.request, that.filters.spotReq);
+    var filtered = sr.request;
```

Comment:
I'm curious, why drop the filtering? The filtering is about removing properties right?